src/components/User/CUserEditorPassword.vue

```diff
@@ -45,6 +45,14 @@
     </template>
 
     <template #footer>
+      <confirmation-toggle
+        class="ml-1"
+        cta-class="light"
+        @confirmed="onRemovePassword()"
```

Comment:
When binding an event listener for Vue, you can just do `@confirmed="onRemovePassword"` instead of `@confirmed="onRemovePassword()"`. Both cases work, because Vue defers execution of the later until the event occurs.

When using the later, you can use the `$event` variable to get the emitted event parameters; so `@confirmed="onRemovePassword($event)"` for example. This can be useful when wanting to pass additional parameters to the variable, such as `@confirmed="onRemovePassword("removing", $event)"` for example.

A little tip here.